src/libmagickwand.jl

```diff
@@ -25,117 +25,15 @@ else
-    DestroyPixelWand[]                 = loadsym(:DestroyPixelWand)
-    MagickGetException[]               = loadsym(:MagickGetException)
-    PixelGetException[]                = loadsym(:PixelGetException)
-    MagickExportImagePixels[]          = loadsym(:MagickExportImagePixels)
-    MagickImportImagePixels[]          = loadsym(:MagickImportImagePixels)
-    MagickConstituteImage[]            = loadsym(:MagickConstituteImage)
-    MagickSetImageDepth[]              = loadsym(:MagickSetImageDepth)
-    MagickGetImagesBlob[]              = loadsym(:MagickGetImagesBlob)
-    MagickPingImage[]                  = loadsym(:MagickPingImage)
-    MagickReadImage[]                  = loadsym(:MagickReadImage)
-    MagickReadImageFile[]              = loadsym(:MagickReadImageFile)
-    MagickReadImageBlob[]              = loadsym(:MagickReadImageBlob)
-    MagickWriteImages[]                = loadsym(:MagickWriteImages)
-    MagickWriteImagesFile[]            = loadsym(:MagickWriteImagesFile)
-    MagickGetImageHeight[]             = loadsym(:MagickGetImageHeight)
-    MagickGetImageWidth[]              = loadsym(:MagickGetImageWidth)
-    MagickGetNumberImages[]            = loadsym(:MagickGetNumberImages)
-    MagickNextImage[]                  = loadsym(:MagickNextImage)
-    MagickResetIterator[]              = loadsym(:MagickResetIterator)
-    MagickNewImage[]                   = loadsym(:MagickNewImage)
-    MagickGetImageAlphaChannel[]       = loadsym(:MagickGetImageAlphaChannel)
-    MagickGetImageProperties[]         = loadsym(:MagickGetImageProperties)
-    MagickGetImageProperty[]           = loadsym(:MagickGetImageProperty)
-    MagickGetImageColors[]             = loadsym(:MagickGetImageColors)
-    MagickGetImageType[]               = loadsym(:MagickGetImageType)
-    MagickSetImageType[]               = loadsym(:MagickSetImageType)
-    MagickGetImageColorspace[]         = loadsym(:MagickGetImageColorspace)
-    MagickSetImageColorspace[]         = loadsym(:MagickSetImageColorspace)
-    MagickSetImageCompression[]        = loadsym(:MagickSetImageCompression)
-    MagickSetImageCompressionQuality[] = loadsym(:MagickSetImageCompressionQuality)
-    MagickGetImageTicksPerSecond[]     = loadsym(:MagickGetImageTicksPerSecond)
-    MagickGetImageDelay[]              = loadsym(:MagickGetImageDelay)
-    MagickSetImageDelay[]              = loadsym(:MagickSetImageDelay)
-    MagickSetImageFormat[]             = loadsym(:MagickSetImageFormat)
-    MagickGetImageDepth[]              = loadsym(:MagickGetImageDepth)
-    MagickGetImageChannelDepth[]       = loadsym(:MagickGetImageChannelDepth)
-    PixelSetColor[]                    = loadsym(:PixelSetColor)
-    MagickRelinquishMemory[]           = loadsym(:MagickRelinquishMemory)
-    MagickQueryConfigureOptions[]      = loadsym(:MagickQueryConfigureOptions)
-    MagickQueryConfigureOption[]       = loadsym(:MagickQueryConfigureOption)
+libversion() = v"6.9.10-4"
```

Comment:
Can this be provided by the builder script? Otherwise I worry we will forget to update it and it will get out-of-sync.

Reply:
Sure, there isn't any infrastructure in place in binaryprovider right now, so I figured that any hacky workaround I do would be as bad as this;)

Reply:
As further background, the call to get the version from ImageMagick directly segfaulted, which I was too tired to debug especially since all other calls worked :D So I just assumed it came naturally from the different build process ...

Reply:
Hmm, we've seen that before and it seems to correlate with problematic initialization. Can you post what you tried somewhere?

Reply:
Investigating it right now! Might have found a problematic flag in the configure ;) Seems like there is a flag to make ImageMagick find all configuration files itself instead of having them hardcoded at compile time.